keyboards/cyboard/dactyl/dactyl.c

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2023 Cyboard LLC (@Cyboard-DigitalTailor)
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+#include "dactyl.h"
+
+#ifdef RGB_MATRIX_ENABLE
+led_config_t g_led_config = { {
+    // left
+    { 7,      8,  9,  NO_LED, NO_LED, NO_LED, NO_LED }, \
+    { 6,      5,  10, 16,     22,     28,     34     }, \
+    { NO_LED, 0,  11, 17,     23,     29,     35     }, \
+    { NO_LED, 1,  12, 18,     24,     30,     36     }, \
+    { NO_LED, 2,  13, 19,     25,     31,     37     }, \
+    { NO_LED, 3,  14, 20,     26,     32,     38     }, \
+    { NO_LED, 4,  15, 21,     27,     33,     39     }, \
+    // right
+    { 47,     48, 49, NO_LED, NO_LED, NO_LED, NO_LED }, \
+    { 46,     45, 50, 56,     62,     68,     74     }, \
+    { NO_LED, 40, 51, 57,     63,     69,     75     }, \
+    { NO_LED, 41, 52, 58,     64,     70,     76     }, \
+    { NO_LED, 42, 53, 59,     65,     71,     77     }, \
+    { NO_LED, 43, 54, 60,     66,     72,     78     }, \
+    { NO_LED, 44, 55, 61,     67,     73,     79     } \
+}, {
+    { 86, 43 }, { 86, 32 }, { 86, 21 }, { 86, 11 }, { 86, 0 }, { 86, 53 }, { 103, 53 }, { 103, 64 }, { 86, 64 }, { 69, 64 }, { 69, 53 }, { 69, 43 }, { 69, 32 }, { 69, 21 }, { 69, 11 }, { 69, 0 }, { 52, 53 }, { 52, 43 }, { 52, 32 }, { 52, 21 }, { 52, 11 }, { 52, 0 }, { 34, 53 }, { 34, 43 }, { 34, 32 }, { 34, 21 }, { 34, 11 }, { 34, 0 }, { 17, 53 }, { 17, 43 }, { 17, 32 }, { 17, 21 }, { 17, 11 }, { 17, 0 }, { 0, 53 }, { 0, 43 }, { 0, 32 }, { 0, 21 }, { 0, 11 }, { 0, 0 }, { 138, 43 }, { 138, 32 }, { 138, 21 }, { 138, 11 }, { 138, 0 }, { 138, 53 }, { 121, 53 }, { 121, 64 }, { 138, 64 }, { 155, 64 }, { 155, 53 }, { 155, 43 }, { 155, 32 }, { 155, 21 }, { 155, 11 }, { 155, 0 }, { 172, 53 }, { 172, 43 }, { 172, 32 }, { 172, 21 }, { 172, 11 }, { 172, 0 }, { 190, 53 }, { 190, 43 }, { 190, 32 }, { 190, 21 }, { 190, 11 }, { 190, 0 }, { 207, 53 }, { 207, 43 }, { 207, 32 }, { 207, 21 }, { 207, 11 }, { 207, 0 }, { 224, 53 }, { 224, 43 }, { 224, 32 }, { 224, 21 }, { 224, 11 }, { 224, 0 }
+}, {
+    // left
+    4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+    4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+    4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+    4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+    // right
+    4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+    4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+    4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+    4, 4, 4, 4, 4, 4, 4, 4, 4, 4
+} };
+#endif
```

Comment:
This should also be migrated to `info.json`, and this file removed.

Reply:
examples of a rgb_matrix layout in `info.json` include `boardsource/lulu` and `crkbd`